src/components/button-toggle-group/button-toggle-group.stories.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import {
   text, number, boolean, select, percentageRange
 } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import { State, Store } from '@sambego/storybook-state';
 import { dlsThemeSelector, classicThemeSelector } from '../../../.storybook/theme-selectors';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import notes from './documentation';
```

Comment:
Shouldnt this file also be removed?

Reply:
I left it as there is another ticket to add the `mdx` file in and some of the content seemed relevant to add there

Reply:
Makes sense :)